Library/Homebrew/utils/puts_columns.rb

```diff
@@ -0,0 +1,41 @@
+require "utils/tty"
+
+module Kernel
```

Comment:
But I think doing that in another PR would be best as it's tricky to review as-is and compare what's changed. Best to leave it as-is in `utils.rb` for now but 👍 to moving it after that.